scripts/reg-form.js

```diff
@@ -219,7 +219,8 @@ $(document).ready(function(){
       },
   
       messages: {
-        r_fname: "This field is required!"
+        r_fname: "This field is required!",
+	r_lname: "This field is required!"
```

Comment:
The validation for r_lname, r_fname were already present. The validation of other fields (like sex, date of birth, email, etc) is required now.